package.json

```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/digitalbazaar/bedrock",
   "dependencies": {
     "async": "^2.6.0",
```

Comment:
I guess there's still more async stuff lingering around?

Reply:
Yes, in logger code.  Not much, but took longer than a few mins to convert so I left it.

Reply:
Ok, fine, converted it.  Removed async and mkdirp deps and bumped Node.js dep to v10.12.0.